picasso/src/main/java/com/squareup/picasso3/Request.java

```diff
@@ -219,7 +211,7 @@ public Builder newBuilder() {
     private float rotationPivotY;
     private boolean hasRotationPivot;
     private boolean purgeable;
-    private List<Transformation> transformations;
+    private List<Transformation> transformations = new ArrayList<>();
```

Comment:
maybe it's alright to let this list be null to signify the empty list? for performance considerations?

Reply:
since lots of Requests get created on a hot path.

Reply:
Yeah we probably need to retain this null pain. Especially since I don't think custom transformations are overwhelmingly common.

Reply:
reverted null removal.